examples/NeuralNetwork-load/sketch.js

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2024 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+// This example will load a pre-trained model that recognizes hand gestures for rock paper scissors.
+
+let classifier;
+let handPose;
+let video;
+let hands = [];
+let classification = "";
+let isModelLoaded = false;
+
+function preload() {
+  // Load the handPose model
+  handPose = ml5.handPose();
+}
+
+function setup() {
+  createCanvas(640, 480);
+
+  // Create the webcam video and hide it
+  video = createCapture(VIDEO);
+  video.size(width, height);
+  video.hide();
+
+  // For this example to work across all browsers
+  // "webgl" or "cpu" needs to be set as the backend
+  ml5.setBackend("webgl");
+
+  // Set up the neural network
+  let classifierOptions = {
+    task: "classification",
+  };
+  classifier = ml5.neuralNetwork(classifierOptions);
+
+  const modelDetails = {
+    model: "model/model.json",
+    metadata: "model/model_meta.json",
+    weights: "model/model.weights.bin",
+  };
+
+  classifier.load(modelDetails, modelLoaded);
```

Comment:
I've got to think about how this can be handled within `preload`.  With the way that I set up the p5 preloading it will work if you include the `modelDetails` as the `modelUrl` property on the `classifierOptions`.  But not if you use the `load` method. 🤔 

I'm playing around with it now and if I move this block into preload I get an error due to not awaiting the `setBackend()` call.  That can be fixed with an `await tf.ready()` in the `neuralNetwork`.

Reply:
Thank you @lindapaiste it would be great to have this in `preload()` as well!